### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Rediska235**

### CODE REVIEW

File: bank system.sql [hunk 1]
```diff
@@ -1 +1,402 @@
...
+/* task 2
+Покажи мне список банков у которых есть филиалы в городе X (выбери один из городов)
+*/
+
+SELECT bank_name
+FROM banks
+    JOIN bank_branches ON bank_id = banks.id
+    JOIN cities ON city_id = cities.id
+WHERE city_name = 'Минск';
+GO
```
Comment (EDanilkov) [inline bank system.sql:124]: Check encoding what you use for creation this database or use english words
Right now I get this response
image

File: bank system.sql [hunk 2]
```diff
@@ -1 +1,402 @@
...
+SELECT s.status_name, 
+       (SELECT COUNT(*)
+        FROM bank_cards 
+            JOIN accounts ON account_id = accounts.id
+            JOIN clients ON client_id = clients.id
+            JOIN social_statuses ON status_id = social_statuses.id
+        WHERE social_statuses.id = s.id) AS card_count
+FROM social_statuses as s
+ORDER BY 1;
+GO
```
Comment (EDanilkov) [inline bank system.sql:173]: I get different result from both queries
image

**Review by EDanilkov — CHANGES_REQUESTED** [review]

File: bank system.sql [hunk 3]
```diff
@@ -1 +1,402 @@
+/* database creation */
+CREATE DATABASE bank_system;
+GO
 
+USE bank_system;
+GO
+
+CREATE TABLE cities(
+    id INT PRIMARY KEY IDENTITY,
+    city_name VARCHAR(30) NOT NULL
```
Comment (EDanilkov) [inline bank system.sql]: Read about naming convention in database
If you will use it in the query, you will have
cities.city_name
Better: Cities.CityName
Much better: Cities.Name

**Review by EDanilkov — CHANGES_REQUESTED** [review]

**Comment by Rediska235** [comment]
Fix encoding errors by switching to English.
Corrected the "name" fields.

File: bank system.sql [hunk 4]
```diff
@@ -1 +1,402 @@
...
+GO
+
+EXEC income_for_social_status 5;
+GO
+
+SELECT client_id, status_id, social_statuses.name AS social_status_name, balance AS account_balance
+FROM accounts
+    JOIN clients ON client_id = clients.id
+    JOIN social_statuses ON status_id = social_statuses.id;
+GO
```
Comment (EDanilkov) [inline bank system.sql:231]: The task is not working, the balance has not changed
image

**Review by EDanilkov — CHANGES_REQUESTED** [review]

File: bank system.sql (hunk 4 above)
Comment (Rediska235) [reply bank system.sql:231]: I don't really understand what the problem is(
I've already checked it on two computers and everything works for me. I don't see any errors in the body of the procedure either.
image

Maybe you accidentally deleted something? Please try again to delete the database and run the script.

**Pull request merged**